app/stac_api/models.py

```diff
@@ -304,7 +320,7 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
         # TODO: check if collection's bbox needs to be updated
         # --> this could probably best be done with GeoDjango? (@Tobias)
         # I leave this open for the moment.
-        self.collection.update_extent(self.properties_datetime)
+        self.collection.update_temporal_extent(self.properties_datetime)
 
         super().save(force_insert, force_update, using, update_fields)
```

Comment:
do you need to hanlde super().delete() as well?

Reply:
Good point, yes, the delete case is still to be implemented.

Reply:
Will do that in a new PR